src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -3949,6 +3949,7 @@ AuctionHouse.SearchTimeout = 1000
 
 Appender.Console=1,4,0,"1 9 3 6 5 8"
 Appender.Server=2,5,0,Server.log,w
+# Appender.Chat=2,4,15,chat_%s.log
```

Comment:
why `%s` ?

Reply:
for date

Reply:
and it will make a new file at every new worldserver start. Same as Appender.GM